src/main/java/com/apicatalog/jsonld/loader/LRUDocumentCache.java

```diff
@@ -0,0 +1,52 @@
+package com.apicatalog.jsonld.loader;
+
+import com.apicatalog.jsonld.JsonLdError;
+import com.apicatalog.jsonld.context.cache.LruCache;
+import com.apicatalog.jsonld.document.Document;
+
+import java.net.URI;
+import java.util.Collection;
+
+public class LRUDocumentCache implements DocumentLoader {
+
+    private final DocumentLoader documentLoader;
+
+    private final LruCache<String, Document> cache;
+
+    public LRUDocumentCache(int cacheSize) {
```

Comment:
Let's drop this constructor. I understand the motivation, but it could in some cases cause confusion. Having only a constructor explicitly asking for a loader to cache helps a developer to realize what is going to be cached. Should be addressed in README by passing `loader` param in the example.